kartothek/io_components/cube/write.py

```diff
@@ -372,6 +401,10 @@ def multiplex_user_input(data, cube):
     return data
 
 
+class MultiTableCommitException(Exception):
```

Comment:
docstring

Reply:
Wasn't that sure about the name of the exception but I felt it pretty convenient during testing to see whether something raised during execution or during a post write check since the latter triggers a rollback, which might be interesting for debugging purposes and might not be obvious to a user. Any thoughts about a good name? Will add a doc string if we keep it, of course

Reply:
`MultiTableCommitAborted` maybe? IMHO sing the word `Exception` or `Error` within the class-name is kinda pointless (except when the class is so abstract that it demands it). Also, I would subclass from `RuntimeError` to be more precise.